stellar/operators.js

```diff
@@ -116,11 +117,7 @@ if (require.main === module) {
     program.name('operators').description('Operators contract management');
 
     program.addOption(
-        new Option('-e, --env <env>', 'environment')
-            .choices(['local', 'devnet', 'stagenet', 'testnet', 'mainnet'])
-            .default('testnet')
-            .makeOptionMandatory(true)
-            .env('ENV'),
+        new Option('-e, --env <env>', 'environment').choices(CHAIN_ENVIRONMENTS).default('testnet').makeOptionMandatory(true).env('ENV'),
```

Comment:
Can you remove this duplication of CLI setup as well? addBaseOptions can be made generic and called everywhere. Maybe you can refactor it on top of this PR to avoid conflicts
https://github.com/axelarnetwork/axelar-contract-deployments/pull/341

Reply:
Approved this one to unblock

Reply:
Yes, will do in a separate PR.